pep-0678.rst

```diff
@@ -330,6 +343,17 @@ does not address the original motivation of compatibility with
 Furthermore, we do not believe that the problem we are solving requires or
 justifies new language syntax.
 
+
+Acknowledgements
+================
+We wish to thank the many people who have assisted us through conversation,
```

Comment:
```suggestion
I wish to thank the many people who have assisted us through conversation,
```

Reply:
In formal and academic publishing circles (at least in AmE), for documents such is this, use of "I" is considered verboten in this context and "we" is preferred even if only a single author is listed. Therefore, from the perspective of a PEP editor, this is a valid choice for a PEP author to use here, if they prefer it, unless there is something I've overlooked specific to this case.

Additionally, this is the voice used throughout the rest of the PEP (including on the line above) so it would be inconsistent to change it here.